index.html

```diff
@@ -5,7 +5,8 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse!</title>
+    <link rel="stylesheet" href="./style.css">
+    <title class="heading">Hello Microverse!</title>
```

Comment:
You did well implementing the previous changes , please use the `h1` tag and not `title` like this 

> 
       <h1 class="heading">Hello Microverse </h1>

Reply:
Ok just saw my silly mistake. Thanks a lot.